controllers/api/comment_routes.js

```diff
@@ -0,0 +1,24 @@
+const router = require("express").Router();
+const { Comment } = require("../../models");
+
+// New Comment rufen und in die Datebank packen.
+router.post("/", async (req, res) => {
+  // Body, da wir es mit der post.js so bestimmt haben.
+  //   try/catch handling bei Fehlerfall.
+  try {
+    const postData = req.body;
+    await Comment.create({
+      comment: postData.comment,
+      user_id: req.session.userId,
+      post_id: postData.postId,
+    });
+    // Es wird immer auf ein res gewartet (req, RES); Status 200 wäre dann hier die RES.
+    // Man muss dem status noch ein Ende mitgeben.
+    res.status(200).end();
```

Comment:
haven't seen the .end() implementation before, nice thinking :)
I'd suggest returning a json response just to reassure whoever would be using your api that everything has gone smoothly.